docs/samples/deploying-your-first-bitcoin-dapp.md

```diff
@@ -23,7 +23,7 @@ and [Bitcoin API](https://internetcomputer.org/docs/current/references/ic-interf
 
 3. Initialize the git submodules
 
-	git submodule update --init --recursive
+        git submodule update --init --recursive
```

Comment:
i may be crazy, but I do not see a difference in the change. Look like identical lines. Can you help me understand @dfx-json ?

Reply:
tabs vs spaces. It actually corrects the formatting.